File: Backend/src/main/java/com/gameshub/controller/DTO/request/ProductRequestDTO.java
package com.gameshub.controller.DTO.request;

import lombok.*;

import java.time.*;
import java.util.List;

@Data
public class ProductRequestDTO {
    private int id;
    private LocalDate dateReceived;
    private String status;
    private String requestType;
    private String title;
    private float price;
    private String description;
    private LocalDate postDate;
    private int count;
    private int sellerId;
    private String category;
    private List<String> images;
}

File: Frontend/src/Controller/DTO/request-dto/ProductRequestDTO.ts
export interface ProductRequestDTO {
    id?: number;
    dateReceived: string;
    status: string;
    requestType: string;
    title: string;
    price: number;
    description: string;
    postDate: string;
    count: number;
    sellerId: number;
    category: string;
    images: string[]; // Array of strings (assuming these are URLs)
}
